docs/index.asciidoc

```diff
@@ -421,6 +422,22 @@ retries are exhausted.
 The amount of time to wait before attempting to retry a failed fetch request
 to a given topic partition. This avoids repeated fetching-and-failing in a tight loop.
 
+[id="plugins-{type}s-{plugin}-sasl_jaas_config"]
+===== `sasl_jaas_config` 
+
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+The plugin local JAAS configuration setting which is not java process shared as setting in `jaas_path` config file. This allows each plugin instance to has its own configuration. If both `sasl_jaas_config` and `jaas_path` configurations are set, the setting here takes precedence.
```

Comment:
Thanks for adding the docs, much appreciated. Couple of suggestions for clarity:

```suggestion
JAAS configuration setting local to this plugin instance, as opposed to settings using config file configured using `jaas_path`, which are shared across the JVM. This allows each plugin instance to have its own configuration. 

If both `sasl_jaas_config` and `jaas_path` configurations are set, the setting here takes precedence.
```